assets/js/contact_us_form_element.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Formats the given date instance with a standard date format.
+ *
+ * @param date Date to format.
+ *
+ * @return Formatted string.
+ */
+function formatDate(date: Date): string {
+  return new Intl.DateTimeFormat(document.documentElement.lang || 'en', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    timeZoneName: 'short',
+  }).format(date);
+}
+
+/**
+ * Replaces a text string anywhere it occurs within an element's children, without resorting to
+ * dangerous innerHTML.
+ *
+ * @param element Element to search.
+ * @param needle Search term.
+ * @param replacement Replacement text.
+ */
+function deepReplace(element: Element, needle: string, replacement: string) {
```

Comment:
Another way to do this would be to create a separate custom element responsible for time formatting, which would also be a bit closer to [similar usage in the IdP](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/time-element) (could maybe even publish & reuse the same code!).

```html
<lg-time data-timestamp="2023-01-20T19:46:02Z" data-format="%{month} %{day}, %{year} at %{hour}:%{minute} %{day_period}">January 20, 2023 at 7:46 PM</lg-time>
```